src/components/Form.js

```diff
@@ -1,14 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+// import { nanoid } from '@reduxjs/toolkit';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
- [ ] Let's try not to have commented unused code in our codebase as they make our code dirty and not professional. Kindly remove all unused codes from your codebase 👍🏽